mhr_api/src/mhr_api/resources/v1/permits.py

```diff
@@ -63,9 +63,14 @@ def post_permits(mhr_number: str):  # pylint: disable=too-many-return-statements
         # Validate request against the schema.
         current_app.logger.debug(f'Extra validation on transport permit json for {mhr_number}')
         # Location may have no street - replace with blank to pass validation
-        if request_json.get('newLocation') and request_json['newLocation'].get('address') and \
-                not request_json['newLocation']['address'].get('street'):
-            request_json['newLocation']['address']['street'] = ' '
+        if request_json.get('newLocation') and request_json['newLocation'].get('address'):
+            if not request_json['newLocation']['address'].get('street'):
+                request_json['newLocation']['address']['street'] = ' '
+            # Location may not have a postal code when in Canada
+            # Address schema validation requires a postal code in Canada.
+            # Add an empty value to pass schema validation.
```

Comment:
I was actually setting the `postalCode` to ' ' so the submission goes through (I saw it in some other submission). Should I remove it for permits?

Reply:
Yes you can remove it after this build. And it is not just permits, but corrections, amendments, and change of location.